policies-guidance/slack-community-guidelines.md

```diff
@@ -0,0 +1,147 @@
+
+
+
+## **🚫 Unacceptable Behavior**
+
+These behaviors are not allowed and will not be tolerated:
+
+* Threats of violence or encouragement of harm.
+* Personal attacks, insults, or inflammatory remarks.
+* Harassment or derogatory comments about someone’s identity or personal attributes (race, gender, sexual orientation, disability, appearance, religion, nationality, political views, etc.).
+* Sharing private information without consent.
+* Impersonating individuals or organizations.
+* Unsolicited direct messages for promotions, services, or recruitment.
+* Spamming channels or individuals.
+* Abusing voice/video calling features without prior consent.
+* Using emojis, GIFs, or reactions to belittle or undermine others.
+
+
+
+## **😄 Thoughtful and Inclusive Use of Emojis**
+
+Emojis can add warmth, clarity, and personality to our conversations. Used thoughtfully, they create a more supportive environment.
+
+### **👍 Encouraged uses**
+
+* Celebrating achievements and milestones: 🎉 ✨ ⭐️
+* Offering encouragement and support: 💪 🙌
+* Expressing gratitude: 🙏 😊
+* Acknowledging agreement or appreciation: 👍 ➕ 💯
+* Task management or progress: 👀 ✅ ❌  
+* Kicking off a thread: 🧵  
+* Pointing to a discussion: 👆👇
+
+### **⚠️ Uses to avoid**
+
+* Using emojis to mock, belittle, or subtly criticize others.
+* Spamming emoji reactions to derail discussions.
+* Using culturally ambiguous or potentially offensive symbols.
+* Replacing necessary, clear feedback with emoji-only responses.
+💡 **Tip:** Always start with clear words, then use emojis to add warmth, not to replace meaning.
+
+
+
+## **💬 Emoji Requests**
+
+We welcome suggestions for new custom emojis that can celebrate and support our community values. Please share requests in \#admin-help and explain how the emoji supports our shared culture. Admins will review all requests.
+
+
+
+## **🤝 Respect Direct Messages & Privacy**
+
+* Only send direct messages if the recipient has indicated they are open to them.
+* Do not use DMs to promote services, events, or job opportunities.  
+* Do not share private DMs without explicit consent, except in cases of harassment or code of conduct violations. 
+
+
+
+## **📞 Voice and Video Calls**
+
+* Always obtain permission before starting a call.
+* If you start a call accidentally, apologize immediately and clarify it was unintentional.
+
+
+
+## **💬 Keep Channels Focused**
+
+* Keep discussions on topic and use the most appropriate channel.
+* Use threads to group related messages and keep channels tidy.
+* Avoid cross-posting the same message in multiple channels.
+* Before starting a new topic, check for existing discussions to which you can contribute.
+
+
+
+## **🛡 Reporting Issues**
+
+If you see or experience behavior that violates these guidelines or the FAIR Code of Conduct:
+
+* Use Slack’s **More actions** menu ("...") on any message to select **Report message**.
+* You can also reach out directly to a Slack admin.
```

Comment:
Add a second sentence with instructions for identifying admins. Maybe something like:

> You can also reach out directly to a Slack admin. If you're unsure who admins are, you can follow the instructions in [this Slack documentation](https://slack.com/help/articles/360003534892-Browse-people-and-user-groups-in-Slack?#h_01HEJH9JJC40KSQBZQDBM7Y5MP) to find admins.